### ISSUE TITLE
Quarantine checks and results management

### ISSUE THREAD

**Issue by JohnMwashuma** [onaio/tally-ho#246]
### Current requirement functionality
---
*This functionality partially exist in the current software.*
### User Story
---
These are results which trigger the quarantine are currently automatically assigned to the audit team, they are quarantine cases. These will be classed separate from archived results, they will be counted under quarantined results. Such case and forms expected will be accounted for in a separate 'bucket' within the counts. The system will alert the administrator if the results in quarantine have the capacity to change the position of the candidates who hold potentially winning positions.

### Proposed Implementation
---

- Ona will require to include an enable and disable for a check on the check workflows.
- Some of quarantined results can be addressed by audit team.
   - Some need to be addressed by supervisor.
   - Some need to be addressed by the commissioner.
- We need to test out the quarantine checks to see what the results would be if we include these results in the total output.
  - System should alert that the difference between the top 2 candidates is actually much lower if the quarantine is set.
     - For example, if quarantined results are included and the commission do not want to announce the preliminary results.
  - If the commission prematurely announces preliminary results, this will trigger allegations of fraud.
  - But if the difference is less than the quarantined result then it is fine to go ahead with the announcement.

### Required Changes
---

**Comment by pld** [comment]
I don't think we need a checkbox. Calculating how quarantined results can affect results is the work to be done here.

**Comment by JohnMwashuma** [comment]
Related to: https://github.com/onaio/tally-ho/issues/245
